src/components/Dialog-Window/Notes/DialogNote.module.scss

```
@@ -13,3 +13,10 @@
   bottom: -1.5rem;
   right: -0.3rem;
 }
+
+.dynamicSavingText {
+  color: gray;
```

The contrast with `gray` foreground on `white` background is too low. Perhaps we can use one of the theme colors here?

```suggestion
  color: var(--theme-color-2);
```

---

I checked `--theme-color-2` for our three themes, and the contrast value is good enough on all of them.

---

Let's forget about this now and do it in an upcoming PR.

---

https://github.com/NDLA-H5P/h5p-topic-map/pull/184

---

Absolutely, looks much better!